### PR TITLE
A little bit of nonsense

### PR DESCRIPTION
**Pull request by zfhui**

### CODE REVIEW

File: app/controllers/zoom/water_cooler_controller.rb [hunk 1]
```diff
@@ -53,12 +61,36 @@ def permitted_params
...
+      "the building",
+      "Sonnenallee",
+      "Neukölln",
+      "Berlin",
+      "planet earth",
+      "the meeting that is harder to get into than Harvard",
+      "the meeting that all successful CTOs attend",
+      "the meeting that most successful people attend",
+      "the meeting that most happy people attend",
+      "the meeting will each them everything about NFTs, ETFs and Blockchain",
```
Comment (zfhui) [inline app/controllers/zoom/water_cooler_controller.rb:77]:
```suggestion
      "the meeting that will each them everything about NFTs, ETFs and Blockchain",
```

**Review by btmo — APPROVED** [review]
🚢

File: app/controllers/zoom/water_cooler_controller.rb [hunk 2]
```diff
@@ -53,12 +61,39 @@ def permitted_params
         )
     end
 
-    def transform_zoom_payload_to_slack_webhook_payload
+    PLACES = [
+      "the Water Cooler meeting",
+      "the cooles Water Cooler meeting",
```
Comment (ovejka) [inline app/controllers/zoom/water_cooler_controller.rb:66]:
```suggestion
      "the coolest Water Cooler meeting",
```

File: app/controllers/zoom/water_cooler_controller.rb [hunk 3]
```diff
@@ -53,12 +61,39 @@ def permitted_params
...
+      "Berghain",
+      "Grießmühle",
+      "the building",
+      "Sonnenallee",
+      "Neukölln",
+      "Berlin",
+      "planet earth",
+      "the Rivera",
+      "the BAE area",
+      "Room 316",
```
Comment (ovejka) [inline app/controllers/zoom/water_cooler_controller.rb:76]:
```suggestion
      "Room 618",
```

Comment (zfhui) [reply app/controllers/zoom/water_cooler_controller.rb:76]: haha... yeah, thanks! I wasn't sure anymore.

**Pull request merged**